lib/data_update_scripts/20200324133751_update_tag_hotness_scores.rb

```diff
@@ -0,0 +1,8 @@
+module DataUpdateScripts
+  class UpdateTagHotnessScores
+    def run
+      # Saving a tag will trigger calculate_hotness_score
+      Tag.find_each(&:save)
```

Comment:
IMO this proposed change is fairly mild in its consequences and we probably don't need to re-save as opposed to letting it run its course naturally.

That's just a thought, I don't know if we _always_ need these scripts on small scoring changes.